### PR TITLE
Streamline installation on Debian Bookworm

### PR DESCRIPTION
**Pull request by hakanardo**
Streamline installation on Debian Bookworm

### CODE REVIEW

**Review by glajsnake — APPROVED** [review]

**Review by glajsnake — APPROVED** [review]

File: constraints.txt
Comment (glajsnake) [inline constraints.txt:1]: Did these come from running
pip-compile --allow-unsafe --output-file=constraints.txt --strip-extras requirements/misc.txt
?

colorama is not here even though it is added to the requirements/misc.txt so I'm suspecting something is missing?

**Review by glajsnake — COMMENTED** [review]
Nice changes!
Just one small clarification needed, please see below.

File: constraints.txt
Comment (hakanardo) [reply constraints.txt:1]: You're probably right (this was a while ago :) I'll recreate the constraints file...

Comment (glajsnake) [reply constraints.txt:1]: Sorry about the review delay!
I'm a bit unused to github, but I realize now that the colorama commit is after this one, so it makes sense that that is not in the constraints file here.

Comment (hakanardo) [reply constraints.txt:1]: No worries! You're right of course, but the second commit should probably update the constraints file as well, right?

Comment (glajsnake) [reply constraints.txt:1]: That would be nice, yes.

**Pull request merged**